### ISSUE TITLE
Fix usage of pylint exceptions

### ISSUE THREAD

**Issue by vkucera** [ginnocen/MachineLearningHEP#662]
`# pylint: disable=` exceptions are overused and misused.
- [ ] Fix warnings that can be fixed.
- [ ] Use the remaining exceptions only within the scope where they are supposed to apply.

**Comment by benedikt-voelkel** [comment]
Indeed @vkucera 
Thanks for bringing that up!
Maybe some explanation for everyone: `pylint` can be used to do different kinds of automatic code (quality) checks, varying from layout to some functional checks.
It is super useful to have `pylint` checking the code so we don't need to all those things by hand. Of course, sometimes, one really wants something to look/work in a very specific way and there are always exceptions to the rules (as usual...): In those cases we use `# pyliint: disable=<warning/error-code>`
However, overusing/misuse makes the entire `pylint` business useless, so only disable if you are absolutely certain about it. If not (or you don't understand a certain `pylint` warning) you can ask me anytime.
Long story short: Whenever you work on a code-snippet in the future, it would be awesome to validate the usage of `# pylint: disable=` statements
